medusa/tv/series.py

```diff
@@ -1554,14 +1555,21 @@ def load_imdb_info(self):
                       {'id': self.indexerid, 'imdb_id': self.imdb_id})
             return
 
+        tmdb_id = self.externals.get('tmdb_id')
+        if tmdb_id:
+            country_code = Tmdb()._get_shows_countries(tmdb_id)
+            countries = '|'
+            for country in country_code.split('|'):
+                countries = countries.join(subtitles.name_from_code(country))
+
         self.imdb_info = {
             'imdb_id': imdb_obj.imdb_id,
             'title': imdb_obj.title,
             'year': imdb_obj.year,
             'akas': '',
             'genres': '|'.join(imdb_obj.genres or ''),
-            'countries': '',
-            'country_codes': '',
+            'countries': '|'.join(countries),
```

Comment:
Make sure you at least leave some comments, stating where this info is coming from.